content/post/how-to-be-bleeding-edge-c++-proposals.md

```diff
@@ -0,0 +1,98 @@
++++
+categories = []
+date = "2015-09-05T23:57:10-07:00"
+description = ""
+keywords = []
+title = "How to be bleeding edge – C++ proposals that'll change the way you code"
+
```

Comment:
You'll want to make sure you include `author = "ccbrown"` and use tags instead of keywords.